contrib/bradfitz/gomemcache/memcache/memcache_test.go

```diff
@@ -89,19 +84,48 @@ func testMemcache(t *testing.T, addr string) {
 				Key:   "key2",
 				Value: []byte("value2"),
 			})
-		assert.NoError(t, err)
+		require.NoError(t, err)
 
 		span.Finish()
 
 		spans := mt.FinishedSpans()
 		assert.Len(t, spans, 2)
 		validateMemcacheSpan(t, spans[0], "Add")
 		assert.Equal(t, span, spans[1])
-		assert.Equal(t, spans[1].TraceID(), spans[0].TraceID(),
-			"memcache span should be part of the parent trace")
+		assert.Equal(t, spans[1].TraceID(), spans[0].TraceID(), "memcache span should be part of the parent trace")
 	})
 }
 
+func TestDeprecatedWrapClient(t *testing.T) {
+	li := makeFakeServer(t)
+	defer li.Close()
+
+	client := WrapClient(memcache.New(li.Addr().String()))
+	client.Timeout = 2 * time.Second // Default timeout is 100ms, it can be short for the CI runner.
```

Comment:
This looks longer, not shorter than 100ms.

I see this below as well in the code that was already here. Can we see what this is supposed to be?